daffodil-core/src/main/scala/org/apache/daffodil/runtime1/ChoiceTermRuntime1Mixin.scala

```diff
@@ -96,20 +91,23 @@ trait ChoiceTermRuntime1Mixin { self: ChoiceTermBase =>
       // get recomputed, but it can appear with a stale value (left-over from parse)
       // in the arriving infoset for unparse.
       //
-      val optDefaultBranch = {
-
+      val optDefaultBranchIfHidden = {
```

Comment:
Naming. The "ifHidden" suffix is misleading. This is the default branch, hidden or not.